doc/geany.txt

```diff
@@ -1645,14 +1642,14 @@ the format::
     name.lang_ext.tags
 
 *lang_ext* is one of the extensions set for the filetype associated
-with the tags. See the section called `Filetype extensions`_ for
+with the tags parser. See the section called `Filetype extensions`_ for
 more information.
 
 
 Default global tags files
 `````````````````````````
 
-For some languages, a list of global tags is loaded when the
+For some languages, global tags files are loaded when the
 corresponding filetype is first used. Currently these are for:
```

Comment:
IMO both versions are not clear at all.  We don't load global tags files files for "some languages", we can load them for any language if it has some.  The only thing the list below lists are the builtin ones (which admittedlt have a different code path in Geany, but the user doesn't need to know).

IMO, if should be around the lines of

> Some global tags files are distributed with Geany and will be loaded automatically when the corresponding filetype is first used.  Currently this includes global tags files for these languages:

And possibly somewhere else write something about tags files being automatically loaded when placed in the right location (if it's not already there).